scripts/psake-build.ps1

```diff
@@ -0,0 +1,190 @@
+##
+##  Psake build file for the Software Entitlement Service SDK
+##
+
+properties {
+    $baseDir = resolve-path ..\
+    $buildDir = "$baseDir\build"
+    $srcDir = resolve-path $baseDir\src
+    $testsDir = resolve-path $baseDir\tests
+    $outDir = "$baseDir\out"
```

Comment:
If this is meant to work on Linux, should be using `Join-Path` for all of these.

Reply:
Good catch.